<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>collimationOptions</class>
 <widget class="QDialog" name="collimationOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>356</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Collimation Overlay Options</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNoPredictiveText</set>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QPushButton" name="addB">
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Create a new Collimation Overlay Element</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="list-add">
         <normaloff>../ekos/auxiliary</normaloff>../ekos/auxiliary</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeB">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Delete the selected Collimation Overlay Element</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="list-remove">
         <normaloff>../ekos/auxiliary</normaloff>../ekos/auxiliary</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QListWidget" name="elementNamesList">
      <property name="minimumSize">
       <size>
        <width>175</width>
        <height>0</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Collimation Overlay Element name (double click to edit)</string>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QGroupBox" name="elementConfigBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Collimation Overlay Element</string>
      </property>
      <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <property name="spacing">
        <number>3</number>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="enableLabel">
         <property name="toolTip">
          <string>Select whether this Element is enabled or not</string>
         </property>
         <property name="text">
          <string>Enable:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QCheckBox" name="EnableCheckBox">
           <property name="text">
            <string/>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="renameB">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Rename</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="7" column="2">
        <widget class="QSpinBox" name="countSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>12</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="nameLabel">
         <property name="toolTip">
          <string>Enter a name for this Collimation Overlay Element. If left empty a name will be generated based on the Type selected.</string>
         </property>
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QComboBox" name="typeComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="colourLabel">
         <property name="toolTip">
          <string>Select color for the Element</string>
         </property>
         <property name="text">
          <string>Color:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="offsetXLabel">
           <property name="text">
            <string>X</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="offsetXSpinBox">
           <property name="minimum">
            <number>-9999</number>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="offsetYLabel">
           <property name="text">
            <string>Y</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="offsetYSpinBox">
           <property name="minimum">
            <number>-9999</number>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="11" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="KColorButton" name="colourButton">
           <property name="toolTip">
            <string/>
           </property>
           <property name="whatsThis">
            <string>Select a color for the Collimation Overlay Element.</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="color" stdset="0">
            <color>
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="thicknessLabel">
           <property name="toolTip">
            <string>Set the line thickness used to draw the element.</string>
           </property>
           <property name="text">
            <string>Thickness:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="thicknessSpinBox">
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="offsetLabel">
         <property name="toolTip">
          <string>Set the Element's offset from its Anchor. For an Anchor Element this is from the center of the image.</string>
         </property>
         <property name="text">
          <string>Offset:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="sizeLabel">
         <property name="toolTip">
          <string>Set the size of the Element</string>
         </property>
         <property name="text">
          <string>Size:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="sizeXLabel">
           <property name="text">
            <string>X</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sizeXSpinBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="value">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="linkXYB">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Link X &amp; Y sizes</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="sizeYLabel">
           <property name="text">
            <string>Y</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sizeYSpinBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="value">
            <number>1000</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="typeLabel">
         <property name="toolTip">
          <string>Select the type of Collimation Overlay Element</string>
         </property>
         <property name="text">
          <string>Type:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="countLabel">
         <property name="toolTip">
          <string>Select number of occurrences of the object type within this element</string>
         </property>
         <property name="text">
          <string>Count:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QSpinBox" name="pcdSpinBox">
           <property name="maximum">
            <number>9999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="rotationLabel">
           <property name="toolTip">
            <string>If there is more than one occurrence set the base rotation angle.</string>
           </property>
           <property name="text">
            <string>Rotation:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="rotationDoubleSpinBox">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="maximum">
            <double>359.899999999999977</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="pcdLabel">
         <property name="toolTip">
          <string>Set the Pitch Circle Diameter where there are multiple occurrences (Count &gt;1)</string>
         </property>
         <property name="text">
          <string>PCD:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="nameLineEdit"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>addB</tabstop>
  <tabstop>removeB</tabstop>
  <tabstop>elementNamesList</tabstop>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>renameB</tabstop>
  <tabstop>EnableCheckBox</tabstop>
  <tabstop>typeComboBox</tabstop>
  <tabstop>sizeXSpinBox</tabstop>
  <tabstop>linkXYB</tabstop>
  <tabstop>sizeYSpinBox</tabstop>
  <tabstop>offsetXSpinBox</tabstop>
  <tabstop>offsetYSpinBox</tabstop>
  <tabstop>countSpinBox</tabstop>
  <tabstop>pcdSpinBox</tabstop>
  <tabstop>rotationDoubleSpinBox</tabstop>
  <tabstop>colourButton</tabstop>
  <tabstop>thicknessSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>collimationOptions</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>collimationOptions</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
